### PR TITLE
Legacy example theme to app-theme

### PR DESCRIPTION
**Pull request by phisch**
## Description
Since we have updated the way themes are created, and we only support the old theme directory out of legacy theme support reasons, the example theme should get updated to the newer way of providing themes. That's what this PR is for.

## Related Issue
https://github.com/owncloud/documentation/pull/2844

## Motivation and Context
Since the Documentation is currently being updated, we should update the theme so the documentation can reference to it as an example.

## How Has This Been Tested?
This is just providing a theme through a different way. Will be tested by hand.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/example-theme/appinfo/info.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<info>
+	<id>example-theme</id>
+	<name>Example OwnCloud Theme</name>
```
Comment (DeepDiver1975) [inline apps/example-theme/appinfo/info.xml]: ownCloud

File: apps/example-theme/appinfo/info.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	<description>This App provides the example OwnCloud theme.</description>
```
Comment (DeepDiver1975) [inline apps/example-theme/appinfo/info.xml]: ownCloud

**Comment by phisch** [comment]
Theme can be enabled as an app now.
Please test and confirm.

File: apps/example-theme/appinfo/info.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	<name>Example ownCloud Theme</name>
+	<description>This App provides the example ownCloud theme.</description>
+	<licence>AGPL</licence>
+	<author>Philipp Schaffrath</author>
+	<version>0.0.1</version>
+	<types>
+		<theme/>
+	</types>
+	<dependencies>
+		<owncloud min-version="9.2" max-version="10" />
```
Comment (DeepDiver1975) [inline apps/example-theme/appinfo/info.xml]: there is no 9.2 - lets use 10 ad min and max

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: apps/example-theme/appinfo/info.xml (hunk 3 above)
Comment (phisch) [inline apps/example-theme/appinfo/info.xml]: Okay, i'll do the same for the enterprise theme.

**Comment by butonic** [comment]
my serverroot is `/home/jfd/www/core` (it is a bind mount for apache with www-data rights)
the webroot is `https://asylum/oc/core/`

But in `\OC\Template\ResourceLocator::appendOnceIfExist` `$file` is `'oc/core/apps/example-theme/core/css/styles.css'` and as a result builds the wrong path: 
`/home/jfd/www/core/oc/core/apps/example-theme/core/css/styles.css` when it should be `/home/jfd/www/core/apps/example-theme/core/css/styles.css`
`OC::$WEBROOT` is correct with `"/oc/core"`

This is a quickfix:
```diff
--- lib/private/Template/ResourceLocator.php (revision )
+++ lib/private/Template/ResourceLocator.php (revision )
@@ -107,8 +107,8 @@
   */
  protected function appendOnceIfExist($root, $file, $webRoot = null) {
 
-  if (substr(ltrim($file, '/'), 0, strlen(\OC::$WEBROOT)) === \OC::$WEBROOT) {
-   $file = substr($file, strlen(\OC::$WEBROOT)-1);
+  if (substr('/'.$file, 0, strlen(\OC::$WEBROOT)) === \OC::$WEBROOT) {
+   $file = substr($file, strlen(\OC::$WEBROOT));
   }
 
   $path = $this->buildPath([$root, $file]);
```

But this should obviously be fixed in a better way.

**Review by DeepDiver1975 — APPROVED** [review]

**Pull request merged**